roles/container_runtime/tasks/docker_restart.yml

```diff
@@ -13,18 +13,12 @@
   - controllers
   - etcd
```

Comment:
Does the shim account for rpm-based etcd?

Reply:
No, to solve the general problem we could either make master-restart restart the systemd unit on standalone hosts or we can differentiate between the two in ansible.

For this particular task I wonder if it's no longer needed. We should restart docker on a static pod host and see if the node properly restarts the etcd pod, I believe it would.

Reply:
Yeah, I don't think we need to bounce etcd here.